guides/common/modules/proc_preparing-the-host-group.adoc

```diff
@@ -0,0 +1,22 @@
+[id="preparing-the-host-group_{context}"]
+
+= Preparing the Host Group
+
+Use this procedure to prepare a host group to convert CentOS 7 machines.
```

Comment:
Before, you spoke of `{RHEL} derivates`.

Reply:
CentOS is a discontinued derivate of RHEL, CentOS 8 being the last one.